includes/Services/FlowService.php

```diff
@@ -0,0 +1,153 @@
+<?php
+namespace NewfoldLabs\WP\Module\Onboarding\Services;
+
+use NewfoldLabs\WP\Module\Onboarding\Data\Options;
+use NewfoldLabs\WP\Module\Onboarding\Data\Flows;
+use NewfoldLabs\WP\Module\Onboarding\Data\Data;
+
+class FlowService {
+
+    public static function initalize_flow_data() { 
+		$flow_data_fixes = Flows::get_fixes();
+        if ( ! ( $flow_data = self::read_flow_data_from_wp_option() ) )
+            $flow_data = self::get_default_flow_data();
+		if ($flow_data_fixes) {
+			foreach($flow_data_fixes as $key=>$value) {
+				$flow_data = self::rename_keys($value, Flows::get_data(), $flow_data);
+				self::update_wp_options_data_in_database($flow_data);
+			}
+		}
+		$flow_data = self::update_flow_data($flow_data);
```

Comment:
If flow data does not exist in the option why are we fixing the default flow data blueprint again?